test/integration/get_records_test.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+// +build mock_integration
+
+package integration
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"github.com/insolar/block-explorer/configuration"
+	"github.com/insolar/block-explorer/etl/connection"
+	"github.com/insolar/block-explorer/test/heavymock"
+	"github.com/insolar/block-explorer/testutils"
+	"github.com/insolar/insolar/insolar/gen"
+	"github.com/insolar/insolar/ledger/heavy/exporter"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type integrationTest struct {
+	suite.Suite
+	s      *testutils.TestGRPCServer
+	c      *connection.MainnetClient
+	i      *heavymock.ImporterClient
```

Comment:
heavymock.ImporterClient is some of connection.MainnetClient

Reply:
let's rename `connection.MainnetClient` to `connection.GrpcClientConnection`?

Reply:
renamed